### DISCUSSION TITLE
Minimum screen size - T6E units supported?

### DISCUSSION THREAD

**Discussion by cregeland** [music-assistant/support#1350] [Q&A] [answered]
Hi :)

I´m considering a couple of T6E units for use as Home Assistant Wall panels (sideloading). I would like to control music on the display (spotify/squeezebox/lms system).

Will the screen size of the T6E unit work with Music Assistant? The T6E display is 4inches - 480x480 pixels (more info [here](https://blakadder.com/moes-t6e/))

Thanks :)

**Comment by cregeland** [comment]
Any input would be greatly appreciated regarding this topic. Thanks :)

**Comment by marcelveldt** [comment, marked as answer]
<img width="522" alt="Scherm­afbeelding 2023-07-23 om 18 48 21" src="https://github.com/music-assistant/support/assets/6389780/3b283f24-56c6-463e-9a41-02a3e497a65f">

the MA interface is fully responsive so will adjust to the screensize. This is how it will look like on 480x480

You can easily test this by using the developer tools of the chrome browser and enable the mobile emulator

**Reply by cregeland** [reply]
Thank you for the quick reply! 👍
